### ISSUE TITLE
label call out lines

### ISSUE THREAD

**Issue by Forna73** [qgis/QGIS#39996]
On Qgis 3.10.8 on Windos 10...

...creating a label and selecting the call out line, if I select  "Manhattan lines" style and Arrow as "kind of vector symbol" un-checking the box "repeat arrow on each segment" the final line is a normal line without the angle. If I leave the box checked the line is represented correctly with the arrow on each segment.

It could be recreated with any new layer.

I also tried with a new profile.

**Comment by gioman** [comment]
@Forna73 this is what I see here, isn't correct?


![Screenshot_20201112_130128](https://user-images.githubusercontent.com/1951107/98943375-49d9fc00-24e7-11eb-9970-4fd1110ce7be.png)

![Screenshot_20201112_130144](https://user-images.githubusercontent.com/1951107/98943384-4cd4ec80-24e7-11eb-94a0-a7cf4916b147.png)

**Comment by roya0045** [comment]
Relevant code section starts from https://github.com/qgis/QGIS/blob/746ee315bd0a4140f6e84617d547cc89103f79aa/src/core/symbology/qgsarrowsymbollayer.cpp#L700

**Comment by Forna73** [comment]
> @Forna73 this is what I see here, isn't correct?
> 
> ![Screenshot_20201112_130128](https://user-images.githubusercontent.com/1951107/98943375-49d9fc00-24e7-11eb-9970-4fd1110ce7be.png)
> 
> ![Screenshot_20201112_130144](https://user-images.githubusercontent.com/1951107/98943384-4cd4ec80-24e7-11eb-94a0-a7cf4916b147.png)

That's correct if you set the arrow properties as "curved" and it works well...
The "Manhattan lines" are lines angled with a 90° angle lile an 'L'

**Comment by Forna73** [comment]
> Relevant code section starts from
> 
> https://github.com/qgis/QGIS/blob/746ee315bd0a4140f6e84617d547cc89103f79aa/src/core/symbology/qgsarrowsymbollayer.cpp#L700

Sorry man... this is ancient aramaic to me...

**Comment by roya0045** [comment]
@Forna73 This was not intended for you directly but anyone willing to maybe fix it.

**Comment by gioman** [comment]
> That's correct if you set the arrow properties as "curved" and it works well...
> The "Manhattan lines" are lines angled with a 90° angle lile an 'L'

@Forna73 I edit the title of this ticket, to be more clear.

**Comment by Forna73** [comment]
I've noticed that this problem persist in 3.16.6, but the option "curved" is disappeared....

**Comment by nyalldawson** [comment]
This is a feature request for new rendering modes in the arrow symbol layer, not a bug.
